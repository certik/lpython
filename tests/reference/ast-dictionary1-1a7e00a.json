{
    "basename": "ast-dictionary1-1a7e00a",
    "cmd": "lpython --show-ast --no-color {infile} -o {outfile}",
    "infile": "tests/dictionary1.py",
    "infile_hash": "0193140223269d4060d60fc088242809fd5942d391157c54dd0e2e0f",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "ast-dictionary1-1a7e00a.stdout",
    "stdout_hash": "85c37b884f8e0fe8752716864e73b0c6052d3e9238b376eaadf81c99",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}