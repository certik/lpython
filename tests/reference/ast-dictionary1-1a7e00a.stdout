(Module [(FunctionDef test_Dict ([] [] [] [] [] [] []) [(AnnAssign (Name x Store) (Subscript (Name dict Load) (Tuple [(Name i32 Load) (Name i32 Load)] Load) Load) () 1) (Assign [(Name x Store)] (Dict [(ConstantInt 1 ()) (ConstantInt 3 ())] [(ConstantInt 2 ()) (ConstantInt 4 ())]) ()) (AnnAssign (Name y Store) (Subscript (Name dict Load) (Tuple [(Name str Load) (Name i32 Load)] Load) Load) () 1) (Assign [(Name y Store)] (Dict [(ConstantStr "a" ()) (ConstantStr "b" ())] [(UnaryOp USub (ConstantInt 1 ())) (UnaryOp USub (ConstantInt 2 ()))]) ()) (AnnAssign (Name z Store) (Name i32 Load) () 1) (Assign [(Name z Store)] (Subscript (Name x Load) (ConstantStr "a" ()) Load) ()) (Assign [(Name z Store)] (Subscript (Name x Load) (ConstantStr "b" ()) Load) ())] [] () ())] [])
